workout-tracker/src/components/MainNavBar/NavigationItems/NavigationItems.js

```diff
@@ -4,7 +4,7 @@ import { NavLink } from 'react-router-dom';
 import './NavigationItems.css';
 
 const navItems = [
-  { id: 'contact', text: 'Contact', link: '/contact', auth: false },
```

Comment:
Wouldn't it be better to comment out this code instead of deleting it?
Also, the 'id' and 'text' on line 7 is 'contact' and 'Home' respectively. Do you think that could be confusing to some?

Reply:
yeah you right